openmetadata-ui/src/main/resources/ui/src/pages/services/index.tsx

```diff
@@ -659,16 +646,14 @@ const ServicesPage = () => {
               </div>
             )}
 
-            {!isNil(paging[serviceName].after) && (
-              <div className="tw-my-4 tw-flex tw-justify-center tw-items-center">
-                <LoadMorePagination
-                  showLoadingText
-                  buttonText="Load more"
-                  handleClick={() => pagingHandler(CursorType.AFTER)}
-                  isLoading={isLoadingMore}
-                  status={status}
-                />
-              </div>
+            {Boolean(
```

Comment:
@ShaileshParmar11 isNil will return boolean so no need to cast it with `Boolean`.